florist/app/jobs/hooks.tsx

```diff
@@ -0,0 +1,10 @@
+import useSWR from "swr";
+import { fetcher } from "../client_imports";
+
+export default function useJobStatus(status: string) {
```

Comment:
I think this function could have a better name, like `getJobsByStatus`

Reply:
Yeah, I thought so as well but since it is a react hook, it has to start with "use". The reason it is only coming up now is because I moved the data fetching outside the status component to decouple data fetching from rendering views which also made it a lot easier to mock the data fetching. So we are stuck having to start the function name with use (I guess thats why swr api is useSWR which from the beginning seemed kind of off naming to me)

Reply:
That naming convention is not cool. But I agree it was better to put it separated, so if it has to start with use wouldn't `useGetJobsByStatus` or maybe `useFetchJobs` or something along those lines be better?

Reply:
Agreed! I switched to the first name you suggested :)